static/_redirects

```diff
@@ -300,6 +300,18 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 # Redirects are case sensitive https://answers.netlify.com/t/case-sensitivity-with-redirects/956/2
 /docs/CHANGELOG /docs/releases/changelog
 
+# Consolidated Reference Page redirects
+# Autocert
+/docs/reference/autocert/autocert-ca /docs/reference/autocert
```

Comment:
IDK how but maybe a splat could cover it all?

Reply:
I thought of a splat here, but the actual result is slightly different. Splat would redirect this path `/docs/reference/autocert/<setting>`  to `/docs/reference/autocert/<some-other-path>`. 

These links are all redirecting to the same page, `/docs/reference/autocert/`, so with a splat, `/docs/reference/autocert/autocert-ca` would redirect to this `/docs/reference/autocertautocert-ca` (this has happened before).